aiida_quantumespresso/parsers/projwfc.py

```diff
@@ -46,7 +46,7 @@ def find_orbitals_from_statelines(out_info_dict):
 
     out_file = out_info_dict['out_file']
     atomnum_re = re.compile(r'atom\s*([0-9]+?)[^0-9]')
-    element_re = re.compile(r'atom\s*[0-9]+\s*\(\s*([A-Za-z]+?)\s*\)')
+    element_re = re.compile(r'atom\s*[0-9]+\s*\(\s*(.*?)\s*\)')
```

Comment:
Instead of matching all characters, could we not just add digits, to be a bit more safe on what is matched?
```suggestion
    element_re = re.compile(r'atom\s*[0-9]+\s*\(\s*([A-Za-z0-9]+?)\s*\)')
```

Reply:
Okay i guess if we want to be strict, we should allow everything that pw.x allows for species labels:

```
X | CHARACTER
-- | --
label of the atom. Acceptable syntax: chemical symbol X (1 or 2 characters, case-insensitive) or chemical symbol plus a number or a letter, as in "Xn" (e.g. Fe1) or "X_*" or "X-*" (e.g. C1, C_h; max total length cannot exceed 3 characters)
```

I'll make a commit with `[A-Za-z0-9_-]`

Reply:
I wouldn't add the hard limit of three characters though. This is an arbitrary limitation they have because of memory allocation, and we have to work around this in some of our projects by updating the source and recompiling it.

Reply:
ah sure, well the latest commit shouldn't impose that. i just added underscores and dashes to your suggestion.